Generator/Templates/Shared/enum.sbntxt

```diff
@@ -2,13 +2,14 @@ using System;
 
 namespace {{ namespace }}
 {
+    {{ include 'header.sbntxt' this ~}}
     {{~ if has_flags ~}}
     [Flags]
     {{~ end ~}}
     public enum {{ name }} : long
     {
-{{~ for member in members
-include 'enum_member.sbntxt' member '        '
-end ~}}
+        {{~ for member in members }}
```

Comment:
This generate an empty line

Reply:
For me enums look fine, e.g.:

```C#
[Flags]
public enum ConnectFlags : long
{

    /// <summary>
    /// ...
    /// </summary>
    After = 1,
    
    /// <summary>
    /// ...
    /// </summary>
    Swapped = 2,
    
}        
```

Reply:
The empty lines I mean are after and before opening and closing braces of the enum:

```cs
[Flags]
public enum ConnectFlags : long
{
    // EMPTY LINE HERE
    /// <summary>
    /// ...
    /// </summary>
    After = 1,
    
    /// <summary>
    /// ...
    /// </summary>
    Swapped = 2,
    // EMPTY LINE HERE
}        
```

But this completely negligible 🙂

Reply:
I think this is okay :smile: